src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -496,6 +506,36 @@
 	text-align: center;
 	text-decoration: none;
 	width: 233px;
+
+	&:active,
+	&:hover {
+		background-color: var(--tec-color-button-primary-hover);
+		color: #fff;
+	}
+
+	&:active {
+		background-color: var(--tec-color-button-primary-active);
+	}
+}
+
+.tec-tickets__admin-settings-tickets-commerce-gateways-item-button-link--active {
+	background-color: #fff;
+	border-color: #3c434a;
+	border-style: solid;
+	border-width: 1px;
+	color: #3c434a;
+
+	&:active,
+	&:hover {
+		background-color: var(--tec-color-button-primary-hover);
+		border-color: var(--tec-color-button-primary-hover);
+		color: #fff;
```

Comment:
We have a variable for white.

Reply:
I do see a variable for a background color, but not for text color. Should I use the BG color for text?

Reply:
@codingmusician we do use it in [some places](https://github.com/the-events-calendar/tribe-common/blob/master/src/resources/postcss/components/full/buttons/_border.pcss#L27), but we can keep the hex value. I'll leave it to you.